docs/spherical-video-v2-rfc.md

```diff
@@ -267,3 +267,155 @@ monoscopic equirectangular video:
               ...
 ```
 
+### WebM (Matroska)
```

Comment:
You may want to modify the italicized intro text too (to mention WebM/Matroska).